apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java

```diff
@@ -55,22 +56,32 @@
  */
 public class PropertySourcesProcessor implements BeanFactoryPostProcessor, EnvironmentAware,
     ApplicationEventPublisherAware, PriorityOrdered {
-  private static final Multimap<Integer, String> NAMESPACE_NAMES = LinkedHashMultimap.create();
+  private static final Multimap<Integer, Multimap<String,String>> APP_NAMESPACE_NAMES = LinkedHashMultimap.create();
   private static final Set<BeanFactory> AUTO_UPDATE_INITIALIZED_BEAN_FACTORIES = Sets.newConcurrentHashSet();
 
   private final ConfigPropertySourceFactory configPropertySourceFactory = SpringInjector
       .getInstance(ConfigPropertySourceFactory.class);
-  private ConfigUtil configUtil;
+  private static ConfigUtil configUtil = ApolloInjector.getInstance(ConfigUtil.class);
```

Comment:
The initial logic is to set up the config util within the `postProcessBeanFactory` method. It's uncertain whether initializing it during the class initialization phase would be effective. For instance, the `apollo.cluster` could be defined in `application.properties` and subsequently assigned to system properties via `ApolloApplicationContextInitializer`.

Reply:
This is a helpless operation. I have retained the default appId method of addNamespaces. addNamespaces is a static method. I need to declare it like this to get the value. Or does addNamespaces get an instance from ApolloInjector every time it is called?